shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.h

```diff
@@ -23,6 +26,40 @@
  */
 - (UIView<UITextInput>*)textInputView;
 
+// UIIndirectScribbleInteractionDelegate
+- (BOOL)indirectScribbleInteraction:(UIIndirectScribbleInteraction*)interaction
+                   isElementFocused:(UIScribbleElementIdentifier)elementIdentifier
+    API_AVAILABLE(ios(14.0));
```

Comment:
Whey are these declared individually instead of implementing the protocol?

Reply:
The code highlighter does wonky things with the API_AVAILABLE annotation in the implementation file, but I will swap to that